### PR TITLE
mwlwifi: Update to latest commit in upstream repo

### PR DESCRIPTION
**Pull request by diizzyy**
Update to latest commit in upstream repo
Bumps 88W8964 firmware to 9.3.0.8

Signed-off-by: Daniel Engberg <daniel.engberg.lists@pyret.net>

### CODE REVIEW

**Comment by lucize** [comment]
for how long ? did you ever encountered this problem before ? https://github.com/kaloz/mwlwifi/issues/225#issuecomment-341172912

**Comment by diizzyy** [comment]
About a week on each, works fine

**Comment by lucize** [comment]
I'll give it a try, I have two phones that make it oops

**Comment by tapper82** [comment]
My 3200acm is running fine with this fermwair too! Uptime of 3 days.

**Comment by lucize** [comment]
neither this one in my case:

```
Thu Nov  2 16:23:34 2017 kern.err kernel: [63833.811785] ieee80211 phy0: cmd 0x9111=SetNewStation timed out
Thu Nov  2 16:23:34 2017 kern.err kernel: [63833.817651] ieee80211 phy0: return code: 0x1111
Thu Nov  2 16:23:34 2017 kern.err kernel: [63833.822208] ieee80211 phy0: timeout: 0x1111
Thu Nov  2 16:23:34 2017 daemon.info hostapd: wlan0: STA 40:40:a7:31:ae:bd IEEE 802.11: deauthenticated due to local deauth request
Thu Nov  2 16:26:51 2017 daemon.notice hostapd: wlan0: AP-STA-POLL-OK a4:e4:b8:xx:xx:xx
Thu Nov  2 16:28:00 2017 cron.info crond[1602]: USER root pid 5305 cmd /etc/checkip
Thu Nov  2 16:31:54 2017 daemon.notice hostapd: wlan0: AP-STA-POLL-OK a4:e4:b8:xx:xx:xx
Thu Nov  2 16:35:00 2017 cron.info crond[1602]: USER root pid 5316 cmd /etc/checkip
Thu Nov  2 16:37:00 2017 daemon.notice hostapd: wlan0: AP-STA-DISCONNECTED a4:e4:b8:xx:xx:xx
Thu Nov  2 16:37:00 2017 daemon.info hostapd: wlan0: STA a4:e4:b8:xx:xx:xx IEEE 802.11: disassociated due to inactivity
Thu Nov  2 16:37:04 2017 kern.err kernel: [64643.201304] ieee80211 phy0: cmd 0x9122=UpdateEncryption timed out
Thu Nov  2 16:37:04 2017 kern.err kernel: [64643.207427] ieee80211 phy0: return code: 0x1122
Thu Nov  2 16:37:04 2017 kern.err kernel: [64643.211985] ieee80211 phy0: timeout: 0x1122
Thu Nov  2 16:37:04 2017 kern.err kernel: [64643.216184] wlan0: failed to remove key (0, a4:e4:b8:xx:xx:xx) from hardware (-5)
Thu Nov  2 16:37:04 2017 daemon.info hostapd: wlan0: STA a4:e4:b8:xx:xx:xx IEEE 802.11: deauthenticated due to inactivity (timer DEAUTH/REMOVE)
Thu Nov  2 16:37:08 2017 kern.err kernel: [64647.228582] ieee80211 phy0: cmd 0x9125=BAStream timed out
Thu Nov  2 16:37:08 2017 kern.err kernel: [64647.234013] ieee80211 phy0: return code: 0x1125
Thu Nov  2 16:37:08 2017 kern.err kernel: [64647.238561] ieee80211 phy0: timeout: 0x1125
Thu Nov  2 16:37:12 2017 kern.err kernel: [64651.242524] ieee80211 phy0: cmd 0x9125=BAStream timed out
Thu Nov  2 16:37:12 2017 kern.err kernel: [64651.247948] ieee80211 phy0: return code: 0x1125
Thu Nov  2 16:37:12 2017 kern.err kernel: [64651.252505] ieee80211 phy0: timeout: 0x1125
Thu Nov  2 16:37:16 2017 kern.err kernel: [64655.256479] ieee80211 phy0: cmd 0x9125=BAStream timed out
Thu Nov  2 16:37:16 2017 kern.err kernel: [64655.261911] ieee80211 phy0: return code: 0x1125
Thu Nov  2 16:37:16 2017 kern.err kernel: [64655.266459] ieee80211 phy0: timeout: 0x1125
Thu Nov  2 16:37:20 2017 kern.err kernel: [64659.270440] ieee80211 phy0: cmd 0x9125=BAStream timed out
Thu Nov  2 16:37:20 2017 kern.err kernel: [64659.275890] ieee80211 phy0: return code: 0x1125
Thu Nov  2 16:37:20 2017 kern.err kernel: [64659.280465] ieee80211 phy0: timeout: 0x1125
Thu Nov  2 16:37:24 2017 kern.err kernel: [64663.284400] ieee80211 phy0: cmd 0x9111=SetNewStation timed out
Thu Nov  2 16:37:24 2017 kern.err kernel: [64663.290267] ieee80211 phy0: return code: 0x1111
Thu Nov  2 16:37:24 2017 kern.err kernel: [64663.294815] ieee80211 phy0: timeout: 0x1111
Thu Nov  2 16:37:24 2017 daemon.notice hostapd: nl80211: nl80211_recv_beacons->nl_recvmsgs failed: -5
Thu Nov  2 16:37:28 2017 kern.err kernel: [64667.301151] ieee80211 phy0: cmd 0x8050=broadcast_ssid_enable timed out
Thu Nov  2 16:37:28 2017 kern.err kernel: [64667.307709] ieee80211 phy0: return code: 0x0050
Thu Nov  2 16:37:28 2017 kern.err kernel: [64667.312268] ieee80211 phy0: timeout: 0x0050
Thu Nov  2 16:37:32 2017 kern.err kernel: [64671.317097] ieee80211 phy0: cmd 0x8127=SetInformationElements timed out
Thu Nov  2 16:37:32 2017 kern.err kernel: [64671.323749] ieee80211 phy0: return code: 0x0127
Thu Nov  2 16:37:32 2017 kern.err kernel: [64671.328296] ieee80211 phy0: timeout: 0x0127
Thu Nov  2 16:37:36 2017 kern.err kernel: [64675.334244] ieee80211 phy0: cmd 0x801c=80211RadioControl timed out
Thu Nov  2 16:37:36 2017 kern.err kernel: [64675.340462] ieee80211 phy0: return code: 0x001c
Thu Nov  2 16:37:36 2017 kern.err kernel: [64675.345011] ieee80211 phy0: timeout: 0x001c
Thu Nov  2 16:37:40 2017 kern.err kernel: [64679.348276] ieee80211 phy0: cmd 0x8126=SetFixedRate timed out
Thu Nov  2 16:37:40 2017 kern.err kernel: [64679.354054] ieee80211 phy0: return code: 0x0126
Thu Nov  2 16:37:40 2017 kern.err kernel: [64679.358602] ieee80211 phy0: timeout: 0x0126
```

**Comment by hnyman** [comment]
There is already a newer commit since a few hours. 
https://github.com/kaloz/mwlwifi/commit/cbb631e06a7c3d88f3e5ad44cb56129278899252

**Comment by lucize** [comment]
the last update doesn't help to wifi stability, in my case, but at least now the kernel won't panic anymore

**Comment by mkresin** [comment]
@lucize Is the current version used in LEDE stable for you? Means, do we break something if we merge this PR?

**Comment by lucize** [comment]
I don't think driver version has something to do with it, is clear that qualcomm chipsets are making the WiFi firmware AC to hang when you reconnect to the wireless, but only after is losing the signal, not by manual disconnect and reconnect
The instabilities where observed after the wireless back port update from 4.14rc, but maybe there were before and didn't noticed because I had a Huawei p10 lite in test and it didn't triggered the bug, after the backport I restarted using the BlackBerry Priv that is triggering it between 4h to 2 days, didn't had time to play around the router range to trigger it "manually"

[here](https://github.com/kaloz/mwlwifi/issues/225#issuecomment-344931271) is an updated firmware that still hangs, but the 5Ghz speed improved a lot (~200mb/s on N device, didn't had that speed on any device until now)
the last driver https://github.com/kaloz/mwlwifi/commit/466368f9454250c2bc024795600d92564553d9bb has some debug that maybe will lead to discovering the problem

 so to sum up, mine hanged with and without this update

**Comment by hauke** [comment]
Thank you for the patch it was added to master.
